src/Components/MyListsHandler.cs

```diff
@@ -288,106 +315,192 @@ private void ResetSearchAndDisplayIngredients(TextBox searchBox, StackPanel ingr
         /// </summary>
         /// <param name="ingredient">The ingredient to display.</param>
         /// <returns>A Border control containing the ingredient row.</returns>
-        private Border CreateIngredientRow(Ingredient ingredient, UserList userList)
+        private Border CreateIngredientRow(Ingredient ingredient, UserList userList, double itemWidth, int ingBoxHeight, StackPanel ingPanel)
         {
-            // Create a row to display ingredient details
-            DockPanel ingredientRow = new DockPanel { Margin = new Thickness(0, 5, 0, 5) };
+            SolidColorBrush ingTextCol = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            SolidColorBrush buttonCol = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            SolidColorBrush borderCol = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+
+            int ingTextFont = 30;
+            int buttonFont = 30;
+
+            double availableWidth = SystemParameters.PrimaryScreenWidth - 200;
 
-            // Ingredient details text
-            TextBlock ingredientText = new TextBlock
+            // Create a DockPanel to display ingredient details
+            DockPanel ingredientRow = new DockPanel
             {
-                Text = $"{ingredient.GetName()} - {ingredient.GetIngType()} | {ingredient.GetAmount()} {ingredient.GetUnit()}",
-                Foreground = textColor,
+                Margin = new Thickness(10),
+                Width = itemWidth,
+                HorizontalAlignment = HorizontalAlignment.Stretch
+            };
+
+            // Ingredient name text
+            TextBlock ingredientNameText = new TextBlock
```

Comment:
Make ingredient name wrap around to the next row. Right now, it just cuts off.
![image](https://github.com/user-attachments/assets/0c52392b-cb7f-40c8-b091-5bc8707d2a90)

Reply:
Same issues as before. I can try and see how it looks.